internal/config/cache.go

```diff
@@ -102,4 +102,7 @@ type RedisCacheConfig struct {
 	MinIdleConn     int           `json:"minIdleConn" mapstructure:"min_idle_conn" yaml:"min_idle_conn"`
 	ConnMaxIdleTime time.Duration `json:"connMaxIdleTime" mapstructure:"conn_max_idle_time" yaml:"conn_max_idle_time"`
 	NetTimeout      time.Duration `json:"netTimeout" mapstructure:"net_timeout" yaml:"net_timeout"`
+	CaCertBytes     string        `json:"-" mapstructure:"ca_cert_bytes" yaml:"-"`
+	CaCertPath      string        `json:"-" mapstructure:"ca_cert_path" yaml:"-"`
+	InsecureSkipTLS bool          `json:"-" mapstructure:"insecure_skip_tls" yaml:"-"`
```

Comment:
One thought I have is maybe we could throw an error in `validate()` if you attempt to supply both bytes and path? Just to help folks from maybe missing this or attempting to supply both by accident and not realizing only the raw bytes are succeeding in being set.

Reply:
it's a good suggestion. I have addressed it.